app/src/main/java/com/android/wardrobeManager/ui/images/ThinHorizontalStripes.java

```diff
@@ -0,0 +1,28 @@
+package com.android.wardrobeManager.ui.images;
+
+import android.graphics.Bitmap;
+
+import java.util.Map;
+
+public class ThinHorizontalStripes implements DesignFilterManager.DesignFilter {
```

Comment:
Instead of having thin and thick horizontal/vertical stripes, it may be better to just pass the thickness in through the constructor. That way, it would make it a lot easier to change them, while also making for less code.